sync.go

```diff
@@ -23,21 +20,14 @@ func (s spec) sync() {
 			fatal("failed to ensure tool dir", err)
 		}
 
-		// Download everything to cache
+		// Download everything to tool directory
 		for _, t := range s.Tools {
 			err = download(t)
 			if err != nil {
 				fatalExec("failed to sync "+t.Repository, err)
```

Comment:
Yeah, cleanup logic would be great. I agree. Probably worth its own revision, though.